scraper/src/merge_datasets.js

```diff
@@ -2,28 +2,162 @@ import fs from "fs";
 import { parse as json2csv } from "json2csv";
 import csv from "csvtojson";
 import permapeopleColumnMapping from "./helpers/column_mapping_permapeople.js";
+import axios from "axios";
+import axiosRetry from "axios-retry";
 import {
   sanitizeColumnNames,
   getSoilPH,
   getHeightEnumTyp,
   getSpreadEnumTyp,
-  fetchGermanName,
+  capitalizeWords,
 } from "./helpers/helpers.js";
 
 /**
- * Fetches the German names for the plants from Wikidata API
+ * Defines the amount of retries we do, if axios encounters errors during a HTTP GET Request.
+ * Increse Delay if we encounter error to prevent 429 Errors.
  */
-const fetchGermanNames = async (plants) => {
-  return Promise.all(
-    plants.map(async (plant) => {
-      if (plant["common_name_de"]) {
-        return plant;
+axiosRetry(axios, {
+  retries: 5, // number of retries
+  retryDelay: (retryCount) => {
+    console.log(`retry attempt: ${retryCount}`);
```

Comment:
It might be worth to consider which HTTP error code was returned.
E.g. we don't need to increase the delay if any other error than 429 comes back.

Similarly, if the target sends status like 404, 401, etc. there is no point in sending another request.
If errors happen often enough this might reduce the scrapers runtime significantly.

(You may ignore this comment if this isn't possible for whatever reason ;) )

Reply:
@badnames Thx for the reviews, they are appreciated. :heart: 

Maybe this can be implemented at one place for all scrapers? It is quite annoying to get a timeout during scraper runs.

Reply:
I implemented it for now to retry only if there is a 429 error, but the other scrapers may need to be reworked so that this would be applied correctly.